client/llb/llbbuild/llbbuild.go

```diff
@@ -92,6 +92,7 @@ func (b *build) Inputs() []llb.Output {
 }
 
 type BuildInfo struct {
+	llb.OpMetadata
 	DefinitionFilename string
```

Comment:
Can we move DefinitionFilename to OpMetadata and eliminate BuildInfo?

Reply:
I'm not sure why? `OpMetadata` is something that can be defined on any vertex, `DefinitionFilename` is a property specific to the build operation and gets added to the proto definition.